### PR TITLE
Update Host Watchdog timeout event name

### PR DESCRIPTION
**Pull request by deepakala-k**
Update host watch dog timeout event name to match in both hostboot-dump-collection enabled and disabled state.
Move the configuration file generation outside the executable block

Upstream links:
https://gerrit.openbmc.org/c/openbmc/openpower-debug-collector/+/62812
https://gerrit.openbmc.org/c/openbmc/openpower-debug-collector/+/62813

Test:
Tested and creates the expected PEL.

### CODE REVIEW

**Comment by geissonator** [comment]
add to approvelist

**Comment by geissonator** [comment]
Please provide link to corresponding upstream changes.

**Comment by deepakala-k** [comment]
> Please provide link to corresponding upstream changes.

https://gerrit.openbmc.org/c/openbmc/openpower-debug-collector/+/62812
https://gerrit.openbmc.org/c/openbmc/openpower-debug-collector/+/62813

File: meson.build
Comment (deepakala-k) [reply meson.build:107]: Just fixed it. Worked on devtool and added the changes here. My bad!

**Comment by geissonator** [comment]
I think at this point, lets just get the general commit message comments addressed upstream and then we can refresh this and get it all merged. https://gerrit.openbmc.org/c/openbmc/phosphor-logging/+/62810 will also need to be merged upstream and we'll have to ask Reed to bump to it.

**Comment by deepakala-k** [comment]
The third commit does not have respective upstream patch, as dump code is not upstream yet.

**Review by devenrao — APPROVED** [review]

**Review by geissonator — APPROVED** [review]
Looks good to me. @ojayanth is our current upstream maintainer on this repo so be good to have him approve here before merge.

**Comment by deepakala-k** [comment]
> Since code isn't merged upstream yet, I bet you can just comment on https://gerrit.openbmc.org/c/openbmc/openpower-debug-collector/+/42328/36/dump/dist/meson.build and note that the conf variable should be renamed.

Done. Thank you!

**Pull request merged**
